.vscode/settings.json

```diff
@@ -13,5 +13,10 @@
   "eslint.autoFixOnSave": true,
   "eslint.lintTask.enable": true,
   "[javascript]": { "editor.defaultFormatter": "dbaeumer.vscode-eslint" },
-  "[typescript]": { "editor.defaultFormatter": "dbaeumer.vscode-eslint" }
+  "[typescript]": {
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
+  },
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
```

Comment:
Just today I saw [Eric Elliot sharing this same tip on Twitter](https://twitter.com/_ericelliott/status/1225579728498618368) 😊

Reply:
Right :) I have the same source.